io/include/detray/io/frontend/detector_reader.hpp

```diff
@@ -25,6 +25,36 @@
 
 namespace detray::io {
 
+/// @brief Read detector components from a list of files
```

Comment:
For my understanding: is this a central method that allows a single read with several files?